qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/minio/main.tf

```diff
@@ -11,7 +11,7 @@ resource "helm_release" "minio" {
   repository = "https://charts.bitnami.com/bitnami"
   chart      = "minio"
   # last release that was Apache-2.0
```

Comment:
This comments makes me think a discussion around licenses might need to happen before we merge this.

Reply:
if `9.2.10` is one of the recent releases we might not be able to use that one because they switched their license to AGLPv3, which raises some licensing problems as discussed in #987. I agree that we need to talk about this as well, we might even need to use another service.